### PR TITLE
Center video on current monitor

### LINKED ISSUE #404: Pressing play on downloaded torrent resets Window position.
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
0.3.3
**What operating system and version?**
Windows 10 latest.
**What did you do?**
Press play on downloaded video
**What did you expect to happen?**
Plays video in window, with the window not moving
**What actually happened?**
Plays video correctly, but resets window screen position.

### PR DESCRIPTION
**Pull request by dcposch**
Fixes #404

### CODE REVIEW

**Comment by dcposch** [comment]
@feross want to check that this actually works on multiple monitors? Otherwise I'll check later. Either way we shouldn't merge this until we're sure

File: main/ipc.js [hunk 1]
```diff
@@ -140,6 +140,14 @@ function setBounds (bounds, maximize) {
   // Assuming we're not maximized or maximizing, set the window size
   if (!willBeMaximized) {
     log('setBounds: setting bounds to ' + JSON.stringify(bounds))
+    if (bounds.x === null && bounds.y === null) {
+      // X and Y not specified? By default, center on current screen
+      var screen = require('screen')
```
Comment (feross) [inline main/ipc.js]: Should be `electron.screen`. Requiring electron modules is deprecated.

Comment (feross) [inline main/ipc.js]: I fixed this post-merge

**Comment by feross** [comment]
I think it's a little weird that half of the object is calculated on the client and the other in the main process, but we can clean that up later.

**Comment by dcposch** [comment]
The client doesn't know which screen it's on. (We could use `remote` but that does synchronous IPC and is a bit jank.) Conversely the main process doesn't know what size the video is.

**Comment by feross** [comment]
We could have the client just send the size of the video to the main window and let it figure it out. But nevermind. This is not important right now.

**Pull request merged**